gvr-immersivepedia/gvrimmersivepedia/src/main/java/org/gearvrf/immersivepedia/Main.java

```diff
@@ -85,17 +85,18 @@ public void onSwipe() {
 		FocusableController.swipeProcess(mGvrContext);
 	}
 
-	public static void clickOut() {
-		if (mGvrContext.getMainScene().equals(Main.dinosaurScene)) {
-			Main.dinosaurScene.closeObjectsInScene();
-		}
-	}
-
-	public void onPause() {
-		if (null != mediaPlayer) {
-			mediaPlayer.stop();
-		}
-
-		dinosaurScene.onPause();
-	}
+    public static void clickOut() {
+        if (null != dinosaurScene && mGvrContext.getMainScene().equals(Main.dinosaurScene)) {
```

Comment:
LGTM. Somehow the legacy codes use class variable reference Main.dinosaurScene mixedly .